provision/aws/rds/Taskfile.yaml

```diff
@@ -0,0 +1,74 @@
+# https://taskfile.dev
+version: '3'
```

Comment:
Is the idea to eventually use this Taskfile in `rosa-cross-dc/Taskfile.yaml` as well?

Reply:
Not really, it was just easier for me to run locally as the `task` tool complains when I forget to set a variable and the bash script uses a default :smile: 
It can be removed if you want.